Maestro/subscriptions.json

```diff
@@ -83,6 +89,16 @@
           "maestroDelay": "00:05:00"
         }
       ]
+    },
+    {
+      "path": "https://github.com/dotnet/versions/blob/master/build-info/dotnet/cli/rel/1.0.0/Latest_Packages.txt",
```

Comment:
Latest_Packages.txt gets updated by the Windows x64 build right now, since that is what pushes the packages to MyGet.

Is this what you want to respond to?